.github/workflows/pull-request-check-publish.yml

```diff
@@ -58,30 +58,38 @@ jobs:
 
       - name: Load Docker images
         run: |
-          docker load -i che-code.tgz
+          docker load -i che-code-amd64.tgz
+          docker load -i che-code-arm64.tgz
           docker load -i che-dev.tgz
 
       - name: Login to Quay.io
-        uses: docker/login-action@v3
+        uses: docker/login-action@v2
```

Comment:
could you clarify - what the reason of this change - is it related to some problem?

Reply:
No issues,
Probably by mistake